event-bus/api/src/test/java/org/apache/james/events/EventDeadLettersContract.java

```diff
@@ -197,6 +197,14 @@ default void removeShouldThrowWhenGroupIsNull() {
                 .isInstanceOf(IllegalArgumentException.class);
         }
 
+        @Test
+        default void removeAllEventsOfAGroupShouldThrowWhenGroupIsNull() {
```

Comment:
What is the result of calling `remove(GROUPA)` then `groupsWithFailedEvents()` ?

Reply:
It should remove the group (in Cassandra group table) (y)